module/lua/lstrlib.c

```diff
@@ -894,7 +894,7 @@ static void addlenmod (char *form, const char *lenmod) {
   size_t l = strlen(form);
   size_t lm = strlen(lenmod);
   char spec = form[l - 1];
-  strcpy(form + l - 1, lenmod);
+  strlcpy(form + l - 1, lenmod, MAX_FORMAT);
```

Comment:
That's kinda useless (and an argument I've had a lot of times about forcing strlcpy), but could it be rewritten to use strlcat instead?

Reply:
It seems to cut off the last char, then strcat, and add last char back.